main.py

```diff
@@ -313,6 +317,117 @@ async def get_running_experiment() -> Optional[int]:
     return None
 
 
+# pylint: disable=too-many-locals
+def organize_result_directory(result_dir_path: str, rid: str) -> bool:
+    """Organizes the result directory.
+    
+    It performs the following:
+        1. Read the metadata file from the RID directory and remove it.
+        2. Move the modified experiment to the RID directory.
+        3. Find the h5 result file and copy it to the RID directory.
+        4. Dump the metadata on the copied result file.
+        5. Remove the metadata file.
+
+    Args:
+        result_dir_path: The full path of the result directory.
+        rid: The run identifier value of the experiment in string.
+
+    Returns:
+        True if the h5 result file exists.
+        Otherwise, stop organizing the result directory and return False. 
+    """
+    rid_dir_path = posixpath.join(result_dir_path, f"_{rid}/")
+    # read the metadata
+    metadata_path = posixpath.join(rid_dir_path, "metadata.json")
+    try:
+        with open(metadata_path, encoding="utf-8") as metadata_file:
+            metadata = json.load(metadata_file)
+    except OSError:  # no chance of happening
+        return False
```

Comment:
I think leaving a log would be better because it looks inefficient to raise an exception without returning the RID list because of only one invalid file.

Reply:
So I came up with two ways.
1. Use `print()`: It is the simplist way, and it leave the logs in the original log file containing all request logs, too.
2. Use `logging`: It is similar to QIWIS. It creates another log file.

What looks better?

Reply:
Since this is an asynchronous server, I think `logging` would be more stable.
However, for now, you don't have to configure the logger setup since it is out of the scope of this PR.
You can just define `logger = logging.getLogger(__name__)` and use it.